src/MiNET/MiNET/Player.cs

```diff
@@ -1659,7 +1787,8 @@ public virtual void SpawnLevel(Level toLevel, PlayerLocation spawnPoint, bool us
 
 				CleanCache();
 
-				ForcedSendChunk(SpawnPosition);
+				//ForcedSendChunk(SpawnPosition);
```

Comment:
Is there a reason for not doing this anymore?

Reply:
Chunks are often sent before the client can process them, because of this they are lost.
The most correct solution is to send them only after teleportation. After that, they will not go, since MINET thinks that the player has already received them.

Reply:
I think this particular case can be returned, but I'm afraid to cache it, since the player may lose it again.

Reply:
done